kgb.py

```diff
@@ -25,13 +25,22 @@ def recordLinechartDataTimestamp(author):
     usersLinechartData = getUsersLinechartData()
     linechartData = usersLinechartData['linechartData']
     linechartData[author].append(timestamp)
-    print(linechartData)
-    #writeTousersLinechartDataFile(linechartData)
+    createNewDictionary(linechartData)
+
+def createNewDictionary(linechartData):
+    with open("usersLinechartData.json") as jsonFile:
+        data = json.load(jsonFile)
+        newDict2 = data
```

Comment:
Can we think of a better name for this
```suggestion
        newLineChartData = data
```